### DISCUSSION TITLE
Options for the overview logbook.

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#2856] [Ideas / Feature Requests]
It would be nice to have configurable options for the overview logbook to be able to show and hide columns like it is done for the advanced logbook.

For example, I do not have a paid subscription to qrz.com, so this column is always "red" (no QSLs sent or received).
The same situation is with the eqsl.cc -- I use an account there only because the cloudlog can handle it automatically, but the neccesarity to create a huge pile of locations is annoying, so I'm thinking to stop using the eqsl.cc.

![изображение](https://github.com/magicbug/Cloudlog/assets/16667173/02c60048-0cbd-47dd-b3eb-88125b0c18ed)

**Comment by int2001** [comment]
If you don't upload or download QSOs to QRZ.com, why did you enter a QRZ key at Station-Location?
If you remove the key, the QRZ col automatically disappears.

We are working on an option to explicitly hide it. So you have the possibility to choose which arrows should be visible.

Question: Can u explain why are u using soooo much Locations? Perhaps there's another optimizing-potential...

**Comment by R1BLH** [comment]
> why did you enter a QRZ key at Station-Location?

QRZ.com lookups do not work without the API key.

>Can u explain why are u using soooo much Locations? Perhaps there's another optimizing-potential...

Sure. eQSL.cc requires that locations with the same callsign and different gridsquares do not overlap in time. So every weekend when I move to my country house, I have to create a new location on eQSL.cc and in my cloudlog, and I have to create a new location when I return to the city.

![изображение](https://github.com/magicbug/Cloudlog/assets/16667173/03baca52-18f5-489d-85f6-a13669e5cb12)

**Reply by HB9HIL** [reply]
> QRZ.com lookups do not work without the API key.

It does. Remove the API Key from your Station Locations. Then you can edit `application/config/config.php`

In Line 19 set `$config['callbook']` to `"qrz"`
In Line 48 and 49 you can specifiy your qrz.com Login credentials with `$config['qrz_username']` and `$config['qrz_password']`

You don't need an API Key for a Basic Lookup.

**Comment by R1BLH** [comment]
> It does. Remove the API Key from your Station Locations. Then you can edit `application/config/config.php`
> 
> In Line 19 set `$config['callbook']` to `"qrz"` In Line 48 and 49 you can specifiy your qrz.com Login credentials with `$config['qrz_username']` and `$config['qrz_password']`
> 
> You don't need an API Key for a Basic Lookup.

And what about other users of my cloudlog?

**Reply by int2001** [reply]
They participate from the credentials at config.php.
Since they never see them, thats okay.

so 1st problem with QRZ should be solved then.. Correct?

**Reply by R1BLH** [reply]
>so 1st problem with QRZ should be solved then.. Correct?

No. I just removed QRZ.com API key from the active station location, but the QRZ solumn is visible still.
"QRZ.com Logbook Realtime Upload" set to "No".

**Reply by HB9HIL** [reply]
You probably need to remove from all Station Locations....

**Reply by R1BLH** [reply]
OMG... Lets try. :)

**Reply by R1BLH** [reply]
>You probably need to remove from all Station Locations....

SQL is my friend :)
Yes, it works. The QRZ column is now hidden.

Could you please clarify one thing about the lookup from QRZ.com? In case of free user lookup data does not contain the grid square, right?

**Comment by R1BLH** [comment]
> Since they never see them, thats okay.

Does QRZ.com have an limit for lookups?

**Reply by HB9HIL** [reply]
I don't think so

**Reply by phl0** [reply]
It has. At least on the GUI front end. There is a time limit. They will answer with HTTP 429
Too Many Requests.

**Reply by int2001** [reply]
yes. but Cloudlog always uses the credentials stored at config.php for Callbook-Lookup.
the key at station_location is only taken for up/download.

So no worries about the volume.

**Comment by int2001** [comment]
> Sure. eQSL.cc requires that locations with the same callsign and different gridsquares do not overlap in time. So every weekend when I move to my country house, I have to create a new location on eQSL.cc and in my cloudlog, and I have to create a new location when I return to the city.

i understand. that's a big problem at eQSL. We're not able to heal that from Cloudlog.
There will be a version of CL - in the near future - where you can chose which QSL-Types will be shown at Overview.

**Reply by R1BLH** [reply]
> We're not able to heal that from Cloudlog.

I know. This is a one of reasons to stop using the eqsl.cc for me.
